python/tskit/drawing.py

```diff
@@ -117,6 +122,20 @@ def check_x_scale(x_scale):
     return x_scale
 
 
+def check_ticks(ticks, default_iterable):
+    """
```

Comment:
I don't think this comment is correct? Also seems more complicated than necessary - when do we use the "Autocalculated tick mark locations not implemented." error?

Reply:
Good point. I think we *do* want to (eventually) allow e.g. ticks=5 to find roughly 5 equally spaced ticks to use, but I haven't implemented that yet, hence the error to remind us to leave the possibility open. I could simply place that idea in the docstring, though? I like leaving NotImplementedErrors in the code to indicate definite things that need to be done, though.